backend/api/routes/review.py

```
@@ -0,0 +1,529 @@
+                    logger.info(f"Job {job_id}: Preview video generated: {preview_hash}")
+                    span.set_attribute("preview_hash", preview_hash)
+                    span.set_attribute("success", True)
+                    
+                    return {"status": "success", "preview_hash": preview_hash}
+                    
+            except Exception as e:
+                logger.error(f"Job {job_id}: Failed to generate preview video: {e}", exc_info=True)
+                span.set_attribute("error", str(e))
+                return {
+                    "status": "error",
+                    "message": f"Failed to generate preview video: {str(e)}"
```

Ruff (0.14.8)

Use explicit conversion flag

Replace with conversion flag

(RUF010)